src/types/Todo.ts

```diff
@@ -0,0 +1,6 @@
+export interface Todo {
+  id: number;
+  userId: number;
```

Comment:
The 'userId' property is not required according to the task description. Consider removing it. 🧐